core/src/components/Code/Code.component.tsx

```diff
@@ -0,0 +1,39 @@
+// import { useRefCallback } from "hooks/useRefCallback";
+import { useData } from "index";
+import React from "react";
+import { BlockComponentProps } from "types/block.types";
+import AceEditor from "react-ace";
+import "ace-builds/src-noconflict/mode-typescript";
+import "ace-builds/src-noconflict/theme-monokai";
+
+interface CodeProps extends BlockComponentProps {
+    data: string;
+};
+
+export const CodeComponent: React.FC<CodeProps> = ({ index, data }) => {
+    const { setBlockData } = useData();
+
+    const handleChange = (value:any) => {
```

Comment:
Why are you using any? Does the library not provide any types for this?

Reply:
no :)